examples/automatic-labelling/examples/unlabelled-resources/README.md

```diff
@@ -0,0 +1,31 @@
+# Unlabelled Resources
+
+This directory comprises an example Terraform module which invokes the
+[root module](root-module) to provision an automatic labelling system and then
+creates an unlabelled Compute Engine VM instance in order to verify that a
+label is automatically applied.
+
+## Requirements
+
+### APIs
+
+The following APIs must be enabled for the project in which the resources will be provisioned:
+
+- Compute Engine API
+
+The [root module required APIs][root-module-required-apis] must also be
+enabled.
+
+### Roles
+
+The following roles must be assigned to the account which will be provisioning
+the infrastructure:
+
+- Compute Admin
+
+The [root module required roles][root-module-required-roles] must also be
+assigned.
+
+[root-module]: ../../
```

Comment:
What is the root module?

Is this perhaps meant as an example for the CFT [Event Function](https://github.com/terraform-google-modules/terraform-google-event-function) module?

Reply:
This was submitted prior to the Event Function module being published. I think this should be closed in favour of the same example provided with said module.